cmake/kokkos_tribits.cmake

```
@@ -79,6 +79,12 @@ MACRO(KOKKOS_PROCESS_SUBPACKAGES)
   if (KOKKOS_HAS_TRILINOS)
     TRIBITS_PROCESS_SUBPACKAGES()
   else()
+  # AJP & DZP added for RPS - based perf testing
+  if (KOKKOS_ENABLE_TRACKED_TESTING)
+    SET(INFRASTRUCTURE_ONLY ON CACHE BOOL "Only build the RAJAPerf infrastructure,no builtin kernels")
+    ADD_DEFINITIONS("-DRAJAPERF_INFRASTRUCTURE_ONLY")
+	ADD_SUBDIRECTORY(tpls/rajaperf)
```

Please avoid using tabs.
```suggestion
    ADD_SUBDIRECTORY(tpls/rajaperf)
```